src/components/NewConversationDialog/NewConversationDialog.vue

```diff
@@ -136,6 +138,9 @@ const NEW_CONVERSATION = {
 	isDummyConversation: true,
 }
 
+const forcePasswordProtection = getTalkConfig('local', 'conversations', 'force-passwords')
+const supportConversationCreationPassword = hasTalkFeature('local', 'conversation-creation-password')
```

Comment:
May not be needed?

Reply:
May be moved to conversation store and checked there. We still need to do separate requests on old server versions

```js
if (isPublic && hasTalkFeature('local', 'conversation-creation-password')) {
	response = await createPublicConversation(conversationName, password)
} else {
	response = await createPublicConversation(conversationName)
	await setConversationPassword(token, password)
}
```

Reply:
I'd prefer to keep setConversationPassword out of `createGroupConversation` when it's not forced. From API perspective, they are two separate actions so we do here.